src/compiler/typescript_to_rust/ast_utilities/miscellaneous.ts

```diff
@@ -48,7 +48,14 @@ export function getRustTypeNameFromTypeNode(typeNode: tsc.TypeNode): Rust {
         return `()`;
     }
 
-    // TODO possibly type literals?
+    if (typeNode.kind === tsc.SyntaxKind.LiteralType) {
+        const literalTypeNode = typeNode as tsc.LiteralTypeNode;
+
+        if (literalTypeNode.literal.kind === tsc.SyntaxKind.NullKeyword) {
+            return `()`;
+        }
+        // TODO possibly other type literals?
```

Comment:
Could you put this comment outside of this if statement, and/or make another comment to explain that we need support for type literals and literal types? Type literals and literal types I believe are different concepts. I believe `tsc.SyntaxKind.LiteralType` and `tsc.SyntaxKind.TypeLiteral` are not the same thing, I assume a `TypeLiteral` is like an object literal, `{}` with properties. LiteralTypes might be things like null (in this case).

Reply:
I've made a second comment.